assessments/adhd.html

```diff
@@ -89,438 +95,495 @@ <h3 class="mm-survey-results-score"></h3>
-                             <select id="dropdown" class="required" >
-                              <option value="" disabled selected>Select your option</option>
-                               <option value = "Never"> Never </option>
-                               <option value = "Rarely"> Rarely </option>
-                               <option value = "Sometimes "> Sometimes  </option>
-                               <option value = "Often"> Often </option>
-                               <option value = "Very Often">Very Often </option>
-                             </select>
-                           <br>
-                           <br>
-                           <label for="dropdown" id="model" style="width: 90vw;"><i style="color: rgb(255, 0, 0); ;">*</i> 5. I fidget or squirm with my hands or feet when I have to sit down for a long time.  </label>
-                           <select id="dropdown" class="required" >
-                            <option value="" disabled selected>Select your option</option>
-                             <option value = "Never"> Never </option>
-                             <option value = "Rarely"> Rarely </option>
-                             <option value = "Sometimes "> Sometimes  </option>
-                             <option value = "Often"> Often </option>
-                             <option value = "Very Often">Very Often </option>
-                           </select>
-                         <br>
-                         <br>
-                         <label for="dropdown" id="model" style="width: 90vw;"><i style="color: rgb(255, 0, 0); ;">*</i> 6. I feel overly active and compelled to do things, like I’m being driven by a motor. </label>
-                         <select id="dropdown" class="required" >
-                          <option value="" disabled selected>Select your option</option>
-                           <option value = "Never"> Never </option>
-                           <option value = "Rarely"> Rarely </option>
-                           <option value = "Sometimes "> Sometimes  </option>
-                           <option value = "Often"> Often </option>
-                           <option value = "Very Often">Very Often </option>
-                         </select>
-                       <br>
-                       <br>
-                                  
-                                 
-                                   
-                                
-                                 
-                                   </div>
-                                   
+                                    Section 2 </h3>
+                                 <h4>Part A</h4>
+                                 <h6>( Questions 1-6 )</h6>
+
+                                 <form id="adhd-survey-form-section2a">
+
+                                    <div class="input-headings">

```

Comment:
Is there individual HTML for everything? Why isnt it not rendered through a JS? Anyway too late to handle this. Needs to change later